components/application-mgt/org.wso2.carbon.identity.application.mgt/src/test/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementServiceImplTest.java

```diff
@@ -93,8 +93,12 @@ public class ApplicationManagementServiceImplTest extends PowerMockTestCase {
     private static final String SAMPLE_TENANT_DOMAIN = "tenant domain";
     private static final String APPLICATION_NAME_1 = "Test application1";
     private static final String APPLICATION_NAME_2 = "Test application2";
-    private static final String APPLICATION_NAME_1_Filter = "name ew application1";
-    private static final String APPLICATION_NAME_2_Filter = "name co 2";
+    private static final String APPLICATION_NAME_1_FILTER = "name ew application1";
+    private static final String APPLICATION_NAME_2_FILTER = "name co 2";
```

Comment:
check whether we can remove these with data provider or change var name to application_name_filter_2